src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/ValidateHearingService.java

```diff
@@ -67,6 +69,37 @@ public List<String> validateHearingWarnings(FinremCaseDetails caseDetails) {
         return List.of();
     }
 
+    public List<String> validateManageHearingErrors(FinremCaseData caseData) {
```

Comment:
Missing javadocs

Reply:
If issue date or fast track decision are null then why wait till the about to submit event to raise an error about it? Isn't this a waste of the user's time? Plus there is nothing they can do about it.

Reply:
If you're going to check those fields are populated then that validation should happen in the about to start event.